SpanishFilter/sections/css_extended.txt

```diff
@@ -4,6 +4,8 @@
 ! https://github.com/AdguardTeam/AdguardFilters/issues/24801
 yahoo.com#?#ul[class^="Mb(0)"] > li.js-stream-content:has(> div > a[rel="nofollow noopener noreferrer"]+div:contains(Patrocinado))
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/79561
+allfeeds.live#$?##overlay { remove: true; }
```

Comment:
Is there any reason why this element must be removed?
`allfeeds.live###overlay` seems to works fine on my end.

Reply:
Yeah, why should it be removed? I am asking myself.